src/plugin/fieldViews/ProseMirrorFieldView.ts

```diff
@@ -230,14 +235,22 @@ export abstract class ProseMirrorFieldView extends FieldView<string> {
       }
 
       shouldDispatchTransaction = true;
+
       tr = tr.replace(
         diffStart,
         endOfInnerDiff,
         node.slice(diffStart, endOfOuterDiff)
       );
     }
+
+    if (Array.isArray(storedMarks) || storedMarks === null) {
```

Comment:
I think it may still apply – It depends how selectively `update` is called on NodeViews when storedMarks changes! One to test if interested, but it's definitely still non-blocking.

Reply:
Ah yes, true. I'll make it more selective.

Reply:
Now done.